flang/test/Examples/ctofortran.f90

```diff
@@ -0,0 +1,59 @@
+! UNSUPPORTED: system-windows
+! RUN: split-file %s %t
+! RUN: %clang -c %t/cfile.c
+! RUN: %flang -flang-experimental-hlfir cfile.o %t/src.f90
+! RUN: a.out | FileCheck %s
```

Comment:
```suggestion
! RUN: %t/ctofortran | FileCheck %s
```

Reply:
I think it may fail because of the missing "./", but I would suggest a named executable file to make it more explicit.

Reply:
Thanks, Slava.  You're a prince!